### PR TITLE
Update KlipTok.fa-IR.json

### PR DESCRIPTION
**Pull request by MohammadAnsari98**
replacing word for search with nonformal style

### CODE REVIEW

File: KlipTok.fa-IR.json [hunk 1]
```diff
@@ -20,13 +20,13 @@
...
 	// Header
-	"SearchForChannel": "برای یک استریمر بگرد...", // Deprecated
+	"SearchForChannel": "جستجو برای پیداکردن استریمر...", // Deprecated
 	"SearchNotFoundFormat": "{0} پیدا نشد", 
 
 	// Search and Search Results
-	"SearchKlipTok": "جستوجو در کلیپ تاک... ",
-	"SearchAllOfKlipTok": "جستوجو در تمام کلیپ تاک {0}", 
-	"SearchResults": "نتایج جستوجو",
+	"SearchKlipTok": "جستجو در کلیپ تاک... ",
```
Comment (Rshiya) [inline KlipTok.fa-IR.json:27]: as i checked replacing جستوجو with جستجو seems a good idea

File: KlipTok.fa-IR.json [hunk 2]
```diff
@@ -20,13 +20,13 @@
 	"Logout": "خروج",
 
 	// Header
-	"SearchForChannel": "برای یک استریمر بگرد...", // Deprecated
+	"SearchForChannel": "جستجو برای پیداکردن استریمر...", // Deprecated
```
Comment (Rshiya) [inline KlipTok.fa-IR.json:23]: i used informal way of speak cuz it was what mr fritz asked for ... a friendly way to say search

File: KlipTok.fa-IR.json (hunk 1 above)
Comment (csharpfritz) [reply KlipTok.fa-IR.json:27]: Looks good to me

File: KlipTok.fa-IR.json (hunk 2 above)
Comment (csharpfritz) [reply KlipTok.fa-IR.json:23]: I like the less formal speech.  Thanks!

**Comment by csharpfritz** [comment]
Thank you for the update!

**Pull request merged**